packages/graphback-core/src/crud/mappingHelpers.ts

```diff
@@ -93,10 +93,10 @@ export function getRelationFieldName(field: any, type: any) {
   return fieldName;
 }
 
-function lowerCaseFirstChar(text: string) {
+export function lowerCaseFirstChar(text: string) {
```

Comment:
I have seen them as internal implementation. Generally if someone will use this in any plugin it will mean that they do not implement the mappings right, but I guess you have some code that will use it as well for table names etc. so Let's merge